src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2558,7 +2562,12 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
                 {
                     // Constrained calls to methods on enum methods resolve to System.Enum's methods. System.Enum is a reference
                     // type though, so we would fail to resolve and box. We have a special path for those to avoid boxing.
+#if READYTORUN
```

Comment:
Could you please do what I wrote in #6315 instead? The right fix is maybe 20 lines of code, plus an investigation whether we need to mirror a perf fix from CoreCLR that would otherwise be easy to miss.

Reply:
The change you alluded to looks like overall goodness, I can easily work on applying it